### ISSUE TITLE
Firefox Copy as cURL payload works in app but not Code extension

### ISSUE THREAD

**Issue by chief1983** [a85/POSTMan-Chrome-Extension#12261]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Did not take long to find a bug in the extension I just installed, so figured I'd report it.

I was going to debug a locally made GraphQL call so I copied the request as cURL in Firefox.  I used the import feature in the new Code Extension and it seemed to work, but when I hit Send, it returned a syntax error.  The import worked in Postman App on the same machine.  More context and steps are below.

### Steps To Reproduce

1. Copy a request from Firefox with newline charactes in a GraphQL payload like the following (possibly specific to how our Angular client is building its payloads, not sure):

```
curl 'http://localhost:4200/graphql' -X POST -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/116.0' -H 'Accept: application/json, text/plain, */*' -H 'Accept-Language: en-US,en;q=0.5' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Origin: http://localhost:4200' -H 'Connection: keep-alive' -H 'Referer: http://localhost:4200/blue-sip/did-management/2345678901' -H 'Cookie: <SOME AUTH STUFF>' -H 'Sec-Fetch-Dest: empty' -H 'Sec-Fetch-Mode: cors' -H 'Sec-Fetch-Site: same-origin' --data-raw '{"operationName":"opName","variables":{"customerId":1234},"query":"query opName($customerId: Int!) {\n  opName(customerId: $customerId) {\n    id\n    ...SOME MORE STUFF...    }\n}\n"}'
```

2. Import into Postman in the Code extension.  Run the imported payload (which will appear broken as described above) and see something like:

```
{
    "errors": [
        {
            "message": "Syntax Error: Unexpected Name \"n\".",
            "locations": [
                {
                    "line": 1,
                    "column": 2123
                }
            ],
            "extensions": {
                "code": "GRAPHQL_PARSE_FAILED",
                "exception": {
...
```

This is a likely result anyways as the GraphQL shouldn't be parseable in any environment with these characters in it.

### Operating System

macOS

### Postman Version

Postman VS Code Extension Version: 0.6.1

### Postman Platform

Other

### User Account Type

Signed In User

### Additional Context?

I think the problem is related to the newline characters.  The app imported it and converted it to a GraphQL body successfully.  The extension imported it as raw JSON, and the query param is rendered strangely with a bunch of 'n' characters shown.  `"query": "query opName($customerId: Int!) {n  opName(customerId: $customerId) {n    idn    ...`.   See what I get in the steps to reproduce above.

**Comment by theshubhamjoshi** [comment]
Hello @chief1983, thank you for reporting this issue. We have just pushed a fix for this issue in the latest v0.6.2 release. Please update your extension to the latest version and try this flow out!

**Comment by chief1983** [comment]
Confirmed fixed, works great now, thanks for the work on this!
